src/test/java/uk/gov/justice/digital/hmpps/keyworker/services/ReconciliationServiceTest.java

```diff
@@ -167,7 +168,7 @@ void testCheckWhenTransferredOutDeallocationOccurs() {
         assertThat(offenderKw.getDeallocationReason()).isNull();
         assertThat(offenderKw.getExpiryDateTime()).isNull();
 
-        service.checkMovementAndDeallocate(OffenderEvent.builder().bookingId(-1L).movementSeq(1L).build());
+        service.checkMovementAndDeallocate(new OffenderEvent(-1L, 1L, null));
```

Comment:
would normally call constructor fields by name when class converted to kotlin

Reply:
Sounds like a good reason to convert OffenderEvent :smile:

Reply:
`OffenderEvent` has been converted.  The problem is that java code only does method calls by position, not by name.  I would have to convert the `ReconciliationServiceTest` to make use of it though.